proto/lib/livebook_proto/app_deployment_status_type.pb.ex

```diff
@@ -0,0 +1,8 @@
+defmodule LivebookProto.AppDeploymentStatusType do
+  use Protobuf, enum: true, syntax: :proto3, protoc_gen_elixir_version: "0.12.0"
+
+  field :connecting, 0
+  field :preparing, 1
+  field :available, 2
+  field :deactivated, 99
```

Comment:
I didn't find anything related to making the Enum value to be converted to `snake_case`, so I didn't use the Enum best pratices to be upcased.

Thoughts?

Reply:
This is fine to me, we should prefer whatever is good for Elixir. Btw is the a reason why you went with 99 here? 4 should be fine as well, the number does not really matter. :)

Reply:
No specific reason, I was thinking about having more status in the future, but doesn't matter, since we didn't allow multiple alias for the same value.